settings.c

```diff
@@ -244,6 +249,26 @@ uint8_t settings_store_global_setting(int parameter, float value) {
     case 40: settings.mag_gap_limit = value; break;
     case 41: settings.mag_gap_enabled = value; break;
     case 42: settings.use_load_cell = value; break;
+    case 46: case 47: case 48: case 49:
+      if (value > TWO_SIXTY_FOURTH) {
+        return(STATUS_INVALID_STATEMENT);
+      }
+      switch (parameter) {
+        case 46:
+          settings.x_microsteps = value;
+          break;
+        case 47:
+          settings.x_microsteps = value;
+          break;
+        case 48:
+          settings.x_microsteps = value;
+          break;
+        case 49:
+          settings.x_microsteps = value;
+          break;
+      }
+      motor_drv_init();
```

Comment:
Good point. Once https://github.com/keyme/grbl/pull/115 gets merged, I'll rebase and check `settings.spi_motor_drivers` before I call `motor_drv_init()`. Alternatively, I can check for the spi compile flag, but that's going to change once #115 gets merged in any case. Note: as mentioned in https://github.com/keyme/kiosk/pull/10630, the old drivers still rely on the `microsteps` mask, so this shouldn't affect 3.2 kiosks. 

As you mentioned in a comment in https://github.com/keyme/grbl/pull/115, we should expand the motor driver layer to take care of the non-SPI motor drivers as well, but for now, these settings are only for the new SPI motor drivers.